judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/FunctionsTest.java

```diff
@@ -496,52 +494,120 @@ public void testTime() {
 
-        long asMilliseconds = timestampFunctions.getAsMilliseconds().orElseThrow();
-        long asMilliseconds2 = timestampFunctions.getAsMilliseconds2().orElseThrow();
+        assertEquals(1664409600000L, timestampFunctions.getAsMilliseconds().orElseThrow());
+        assertEquals(1664449871000L, timestampFunctions.getAsMilliseconds2().orElseThrow());
 
-        OffsetDateTime fromMilliseconds = timestampFunctions.getFromMilliseconds().orElseThrow();
+        assertEquals(OffsetDateTime.of(2022, 9, 29, 11, 11, 11, 0, ZoneOffset.UTC), timestampFunctions.getFromMilliseconds().orElseThrow());
 
-        OffsetDateTime plusAll = timestampFunctions.getPlusAll().orElseThrow();
-        OffsetDateTime plusDate = timestampFunctions.getPlusDate().orElseThrow();
+        assertEquals(OffsetDateTime.of(2023, 10, 30, 12, 12, 12, 1000000, ZoneOffset.UTC), timestampFunctions.getPlusAll().orElseThrow());
+        assertEquals(OffsetDateTime.of(2023, 10, 30, 11, 11, 11, 0, ZoneOffset.UTC), timestampFunctions.getPlusDate().orElseThrow());
 
-        OffsetDateTime plusAllReversed = timestampFunctions.getPlusAllReversed().orElseThrow();
-        OffsetDateTime plusDateReversed = timestampFunctions.getPlusDateReversed().orElseThrow();
+        assertEquals(OffsetDateTime.of(2023, 10, 30, 12, 12, 12, 1000000, ZoneOffset.UTC), timestampFunctions.getPlusAllReversed().orElseThrow());
+        assertEquals(OffsetDateTime.of(2023, 10, 30, 11, 11, 11, 0, ZoneOffset.UTC), timestampFunctions.getPlusDateReversed().orElseThrow());
 
-        OffsetDateTime plusMilliseconds = timestampFunctions.getPlusMilliseconds().orElseThrow();
+        assertEquals(OffsetDateTime.of(2022, 9, 29, 11, 11, 11, 1000000, ZoneOffset.UTC), timestampFunctions.getPlusMilliseconds().orElseThrow());
 
-        OffsetDateTime minusAll = timestampFunctions.getMinusAll().orElseThrow();
-        OffsetDateTime minusDate = timestampFunctions.getMinusDate().orElseThrow();
+        assertEquals(OffsetDateTime.of(2021, 8, 28, 10, 10, 9, 999000000, ZoneOffset.UTC), timestampFunctions.getMinusAll().orElseThrow());
+        assertEquals(OffsetDateTime.of(2021, 8, 28, 11, 11, 11, 0, ZoneOffset.UTC), timestampFunctions.getMinusDate().orElseThrow());
 
-        OffsetDateTime minusAllReversed = timestampFunctions.getMinusAllReversed().orElseThrow();
-        OffsetDateTime minusDateReversed = timestampFunctions.getMinusDateReversed().orElseThrow();
+        assertEquals(OffsetDateTime.of(2021, 8, 28, 10, 10, 9, 999000000, ZoneOffset.UTC), timestampFunctions.getMinusAllReversed().orElseThrow());
+        assertEquals(OffsetDateTime.of(2021, 8, 28, 11, 11, 11, 0, ZoneOffset.UTC), timestampFunctions.getMinusDateReversed().orElseThrow());
+
+        assertEquals(OffsetDateTime.of(2022, 9, 29, 11, 11, 10, 999000000, ZoneOffset.UTC), timestampFunctions.getMinusMilliseconds().orElseThrow());
+    }
+
+    @Test
+    public void testTimestampAsString() {
```

Comment:
Create a new entity for this test case in the jsl model, please.

Reply:
https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/146/commits/0d16858f45c94dfde6b5942079927e7bf2753e92